### ISSUE TITLE
`from 10am to now` structures result in incorrect `now`

### ISSUE THREAD

**Issue by alexgorbatchev** [wanasit/chrono#205]
```
> c.parse('from 10am to now')

[
  {
    "ref": "2017-07-03T18:34:16.068Z",
    "index": 0,
    "text": "from 10am to now",
    "tags": {
      "ENTimeExpressionParser": true,
      "ENCasualDateParser": true,
      "ENMergeDateRangeRefiner": true
    },
    "start": {
      "knownValues": {
        "hour": 10,
        "minute": 0,
        "meridiem": 0,
        "day": 3,
        "month": 7,
        "year": 2017
      },
      "impliedValues": {
        "second": 0,
        "millisecond": 0
      }
    },
    "end": {
      "knownValues": {
        "day": 3,
        "month": 7,
        "year": 2017,
        "hour": 10,
        "minute": 0,
        "meridiem": 0
      },
      "impliedValues": {
        "second": 16,
        "millisecond": 68
      }
    }
  }
]
```

It appears that `now` in these type of strings gets the same hour/minute as the left side, but seconds and milliseconds are current time as expected.

**Comment by alexgorbatchev** [comment]
The problem appears to be in `src/refiners/EN/ENMergeDateRangeRefiner.js`. If I short-circuit `parse()` to just return `results`, I get the following results:

```
> c.parse('from 10am to now')

[
  {
    "ref": "2017-07-03T18:59:09.181Z",
    "index": 0,
    "text": "from 10am",
    "tags": {
      "ENTimeExpressionParser": true
    },
    "start": {
      "knownValues": {
        "hour": 10,
        "minute": 0,
        "meridiem": 0
      },
      "impliedValues": {
        "second": 0,
        "millisecond": 0,
        "day": 3,
        "month": 7,
        "year": 2017
      }
    }
  },
  {
    "ref": "2017-07-03T18:59:09.181Z",
    "index": 13,
    "text": "now",
    "tags": {
      "ENCasualDateParser": true
    },
    "start": {
      "knownValues": {
        "day": 3,
        "month": 7,
        "year": 2017
      },
      "impliedValues": {
        "hour": 11,
        "minute": 59,
        "second": 9,
        "millisecond": 181
      }
    }
  }
]
```

**Comment by alexgorbatchev** [comment]
Looking further into this, the following block messes it up for `now`. Commenting it out makes 
`c.parse('from 10am to now')` works correctly. 

```
            for (var key in fromResult.start.knownValues) {
                if (!toResult.start.isCertain(key)) {
                    toResult.start.assign(key, fromResult.start.get(key));
                }
            }
```

**Comment by alexgorbatchev** [comment]
Hm... none of the tests are failing with this block commented out :) Could you please explain the purpose?

**Comment by wanasit** [comment]
Hi. Fixed the problem differently in ce9bb6400fba21d06771d721d765a6438e158d8e.

That for loop inherit known value from the beginning part e.g. "August 12, 2015 to 13 September", the ending should be 13 September 2015.   

It's not that the logic make sense or those cases happen in practice, I just believe that the root cause is how we defining "now" as an uncertain time. 

If the `toResult` is "now". Then, `toResult.start.isCertain('hour')` should return be `true` because as of now it's 9.27 PM (on my time).
